api/src/modules/widgets/widgets.service.ts

```diff
@@ -19,8 +22,10 @@ export class WidgetsService extends AppBaseService<
     protected readonly logger: Logger,
     @InjectRepository(BaseWidget)
     private baseWidgetRepository: Repository<BaseWidget>,
+    @Inject('SurveyDataRepository')
```

Comment:
Please use a token or a Symbol, avoid magic strings whenever possible. If we need this repo injected elsewhere, we have to use another magic string